Makefile

```diff
@@ -24,11 +20,14 @@ check-cmds:
 	./slacker.py --verbose --check
 
 check-style:
-	flake8 --ignore E111,E114,E121,E126,E302 --max-line-length 100 --count --show-source \
+	.venv/bin/flake8 --ignore E111,E114,E121,E126,E302 --max-line-length 100 --count --show-source \
 	  slacker slacker.py
 
 static-analysis:
-	vulture --min-confidence 60 --sort-by-size --exclude slacker/commands/__init__.py \
+	.venv/bin/vulture --min-confidence 60 --sort-by-size --exclude slacker/commands/__init__.py \
 	  slacker slacker.py .vulture_whitelist.py
 
-check: check-cmds check-style static-analysis
+check-min-version:
+	.venv/bin/vermin -t 3.4 slacker slacker.py
+
+check: check-cmds check-style static-analysis check-min-version
```

Comment:
If this is executed in order from left -> right, would it make sense to put the `check-min-version` first (left most)?

Reply:
Yeah, that's a good point. I'll change that.